Package.swift

```diff
@@ -19,7 +19,10 @@ let package = Package(
         .target(
             name: "Loggie",
             dependencies: [],
-            path: "Loggie/Classes"
+            path: "Loggie",
```

Comment:
Will this work as expected (removing `/Classes` from the path)?

Reply:
Yes, since this is the "root" path, including Assets, which is specified in the resources section